posthog/management/commands/test/test_sync_persons_to_clickhouse.py

```diff
@@ -106,6 +114,52 @@ def test_distinct_ids_deleted(self):
         )
         self.assertEqual(ch_person_distinct_ids, [(UUID(int=0), self.team.pk, "test-id-7", 107, True)])
 
+    def test_group_sync(self):
```

Comment:
Can we validate that running `run_group_sync` twice doesn't result in another update?